patches/core/ungoogled-chromium/fix-building-without-safebrowsing.patch

```diff
@@ -349,3 +424,57 @@
  
    // Returns whether the given |origin| is allowed to ask for write access.
    // This is used to block save file dialogs from being shown
+--- a/chrome/browser/browser_resources.grd
++++ b/chrome/browser/browser_resources.grd
+@@ -604,7 +604,6 @@
+       </if>
+       <if expr="not is_android">
+         <include name="IDR_TAB_RANKER_EXAMPLE_PREPROCESSOR_CONFIG_PB" file="resource_coordinator\tab_ranker\example_preprocessor_config.pb" type="BINDATA" />
+-        <include name="IDR_TAB_RANKER_PAIRWISE_EXAMPLE_PREPROCESSOR_CONFIG_PB" file="resource_coordinator\tab_ranker\pairwise_preprocessor_config.pb" type="BINDATA" />
+       </if>
+       <if expr="chromeos">
+         <include name="IDR_SMART_DIM_20181115_EXAMPLE_PREPROCESSOR_CONFIG_PB" file="chromeos\power\ml\smart_dim\20181115_example_preprocessor_config.pb" type="BINDATA" />
+--- a/chrome/browser/resource_coordinator/tab_ranker/tab_score_predictor.cc
++++ b/chrome/browser/resource_coordinator/tab_ranker/tab_score_predictor.cc
+@@ -310,17 +310,6 @@ void TabScorePredictor::LazyInitialize()
+     DCHECK_EQ(preprocessor_config_->feature_indices().size(),
+               static_cast<std::size_t>(tfnative_model::FEATURES_SIZE));
+   }
+-
+-  if (type_ == kPairwiseScorer) {
+-    if (!preprocessor_config_)
+-      preprocessor_config_ = LoadExamplePreprocessorConfig(
+-          IDR_TAB_RANKER_PAIRWISE_EXAMPLE_PREPROCESSOR_CONFIG_PB);
+-    if (!pairwise_alloc_)
+-      pairwise_alloc_ = std::make_unique<pairwise_model::FixedAllocations>();
+-    DCHECK(preprocessor_config_);
+-    DCHECK_EQ(preprocessor_config_->feature_indices().size(),
+-              static_cast<std::size_t>(pairwise_model::FEATURES_SIZE));
+-  }
+ }
+ 
+ // Simply returns the frecency_score in the TabFeatures.
+--- a/chrome/browser/password_manager/chrome_password_manager_client.h
++++ b/chrome/browser/password_manager/chrome_password_manager_client.h
+@@ -30,7 +30,6 @@
+ #include "components/password_manager/core/browser/password_reuse_detection_manager.h"
+ #include "components/password_manager/core/browser/sync_credentials_filter.h"
+ #include "components/prefs/pref_member.h"
+-#include "components/safe_browsing/buildflags.h"
+ #include "content/public/browser/render_widget_host.h"
+ #include "content/public/browser/web_contents_binding_set.h"
+ #include "content/public/browser/web_contents_observer.h"
+@@ -176,13 +175,6 @@ class ChromePasswordManagerClient
+           saving_flow_recorder);
+ #endif  // defined(OS_ANDROID)
+ 
+-#if defined(ON_FOCUS_PING_ENABLED)
```

Comment:
I'm also wondering if `ON_FOCUS_PING_ENABLED` is set by a GN flag too

Reply:
I addressed this question in the review above.